packages/node_modules/@webex/internal-plugin-calendar/test/integration/spec/calendar.js

```diff
@@ -89,7 +89,7 @@ function postToWhistler(webex, type, payload) {
     }));
 }
 
-describe.skip('plugin-calendar', () => {
+describe('plugin-calendar', () => {
```

Comment:
Are these tests working now? Why are they no longer skipped and part of this meetings PR?

Reply:
pulled the old branch so got messed up